pkg/netutil/isolate_linux.go

```diff
@@ -41,6 +41,82 @@ func RecoverPort(port int) error {
 	return err
 }
 
+// SetPacketCorruption corrupts packets at p%
+func SetPacketCorruption(p int) error {
```

Comment:
most of this will be corrected by tcp checksums, for the packets that aren't, I don't see how etcd would be able to pass its checks (e.g., suppose a lease key is corrupted and when the lease checker looks for the intended key, it's gone)